src/backend/distributed/sql/downgrades/citus--10.0-4--9.5-1.sql

```diff
@@ -1,4 +1,44 @@
--- citus--10.0-1--9.5-1
+-- citus--9.5-1--10.0-4
+
+-- This migration file aims to fix the issues with upgrades on clusters without public schema.
+
+-- This file is created by the following command, and some more changes in a separate commit
+-- cat citus--10.0-3--10.0-2.sql citus--10.0-2--10.0-1.sql citus--10.0-1--9.5-1.sql > citus--10.0-4--9.5-1.sql
```

Comment:
although 10.0-4--10.0-3 is empty, the comments in there might be beneficial to keep. Do you think it makes sense to add that one as well, marked with the `copy of` header as well?